O/OSRM/build_tarballs.jl

```diff
@@ -0,0 +1,145 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "OSRM"
```

Comment:
Overly terse for something specific.

```suggestion
name = "OpenStreetRoutingMachine"
```

Reply:
Alternatively, OpenStreetMapOSRM ?

Does OSRM work on something other than OpenStreetMap data?

Does the OSRM use of the term “routing” generalise to (mathematical) graphs, networks etc. or is it geo/street map specific?

Though the source/upstream/official naming might be X, I do not think it automatically justifies this name (X) in the Julia General registry - if X only makes sense in a certain context.

Reply:
But OpenSourceRoutingMachine is also sufficiently specific/verbose to be fine :-)

(Side note: Why are GEOS, MEOS, OSRM naming their projects OpenSourceX ?)

Reply:
(Side note: Why are GEOS, MEOS, OSRM naming their projects OpenSourceX ?)